celesta-sql/src/test/java/ru/curs/celesta/score/GrainModelTest.java

```diff
@@ -9,14 +9,12 @@
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
-import ru.curs.celesta.CelestaException;
-
 public class GrainModelTest {
 
 	private AbstractScore s = new CelestaSqlTestScore();
 
 	@BeforeAll
-	public static void Setup() throws CelestaException {
+	public static void Setup()  {
```

Comment:
this should be lowercase, just as Codacy suggests

Reply:
(I mean, 'setup' instead of 'Setup')

Reply:
fixed